pages/implementation/devs-101.mdx

```diff
@@ -0,0 +1,286 @@
+  There is a third advanced mixed option, where you provide an initialized SDK to Avo, <Link passHref href="/help/troubleshooting"><a>contact us</a></Link> if you want to explore it.
+- `debugger` - mobile platforms (Android, iOS, React Native) support <Link passHref href="/explore-tracking-plan/start-using-visual-debuggers"><a>the Avo Analytics Debuggers</a></Link>, provide one here if you want to connect it to Avo events. You don't need to provide the debugger on the web to use it there.
+- `strict` - if this is set to true Avo would crash your app in the development environment when it sees errors.
+- `noop` - if this flag is set to true Avo won't make any network calls (no tracking) in development and staging environments.
+- `destination options` (not shown in the example) - some platforms, for example Javscript, allow you to provide destination options for the analytics destinations that will be passed during tracking SDK initialization.
+
+### <a name="avo-generated-file-init-inside"></a> What's inside the initAvo method / Avo constructor?
+
+There are a few things happening, like setting the flags, system properties and reporting function invocation in development mode to the Avo servers, so you can check whether the function has ever been successfully invoked in development on the Avo website.
+
+But the most important part is initialization of the analytics destinations. Depending on the setup of your tracking plan you might see something like
+
+```swift
+if __ENV__ == .prod {
+    amplitudeDestination = AmplitudeDestination()
+    amplitudeDestination?.make("production Amplitude key is automatically here based on the data you provided in the tracking plan"")
+    mixpanelDestination = MixpanelDestination()
+    mixpanelDestination?.make("production Mixpanel key is automatically here based on the data you provided in the tracking plan"")
+}
+if __ENV__ == .dev {
+    amplitudeDestination = AmplitudeDestination()
+    amplitudeDestination?.make("development Amplitude key is automatically here based on the data you provided in the tracking plan"")
+    mixpanelDestination = MixpanelDestination()
+    mixpanelDestination?.make("development Mixpanel key is automatically here based on the data you provided in the tracking plan"")
+}
+```
+
+The keys are set up in the Avo web UI and automatically fetched by the codegen, so you don't need to do anything with this, just know that it is there.
+
+### Avo Functions
+
+Avo functions are actual callable functions generated for events defined in your tracking plan.
+
+> On some server platforms Avo functions are async, i.e. on Node.js they return a Promise and on C# you can await on the Avo Functions
+
+For example if you have an event defined with name `Track Event with String Parameters` connected to your source, the Avo functions interface will look like
```

Comment:
Should we use more relateable name here? This might be a bit too meta for some people 😅 
```suggestion
For example if you have an event defined with name `App Opened` connected to your source, the Avo functions interface will look like
```